### PR TITLE
Mt 32014 release 8

### PR DESCRIPTION
**Pull request by chlulei**
Addresses mantis ticket 32014: https://mantis.ilias.de/view.php?id=32014

### CODE REVIEW

File: Modules/Course/classes/class.ilObjCourseAdministrationGUI.php [hunk 1]
```diff
@@ -39,4 +42,31 @@ protected function getAdministrationFormId(): int
     {
         return ilAdministrationSettingsFormHandler::FORM_COURSE;
     }
+
+    protected function addChildContentsTo(ilPropertyFormGUI $form): ilPropertyFormGUI
+    {
+        $checkBox = new ilCheckboxInputGUI($this->lng->txt('crs_my_courses_groups_enabled'), self::SETTING_COURSES_AND_GROUPS_ENABLED);
+        $checkBox->setInfo($this->lng->txt('crs_my_courses_groups_enabled_info'));
+        $checkBox->setChecked((bool) $this->settings->get(self::SETTING_COURSES_AND_GROUPS_ENABLED, 0));
```
Comment (smeyer-ilias) [inline Modules/Course/classes/class.ilObjCourseAdministrationGUI.php]: Please set the default to "1"

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]
Thanks @chlulei, please fix the default value

**Comment by smeyer-ilias** [comment]
Thanks @chlulei!

**Comment by iszmais** [comment]
@chlulei Within the procession of the ticket i stumbled upon 1 criteria:
```Default status for both new settings is enabled.```
And since a default should be enforced by the providing source and not the the demanding i would appreciate if you could also add an update step to set such default, just like @alex40724 did in the similar setting: https://github.com/ILIAS-eLearning/ILIAS/commit/28962f2945712d231c94cb322f0e39e3ec9b6e13

**Pull request merged**